src/components/scenes/FioAddressSettingsScene.js

```diff
@@ -39,11 +39,37 @@ type Props = StateProps & DispatchProps & ThemeProps & OwnProps
 
 class FioAddressSettingsComponent extends React.Component<Props, LocalState> {
   state: LocalState = {
+    showAddBundles: false,
     showTransfer: false
   }
 
   componentDidMount(): * {
-    this.props.refreshAllFioAddresses()
+    const { refreshAllFioAddresses, route } = this.props
+    const { showAddBundles } = route.params
```

Comment:
Please change all variables that have `bundles` to `bundledTxs` or `bundled_txs` bundles is not a proper name